lib/commonConfig.js

```diff
@@ -91,7 +91,8 @@ function processEnvironmentVariables() {
              'IOTA_APPEND_MODE',
              'IOTA_POLLING_EXPIRATION',
              'IOTA_POLLING_DAEMON_FREQ',
-             'IOTA_MULTI_CORE'
+             'IOTA_MULTI_CORE',
+             'IOTA_JSON_LD_CONTEXT'
```

Comment:
New env var should be described in documentation.

Reply:
Fixed b41a18a  - Minimal Documentation for now.